src/Helpers/Retry.ts

```diff
@@ -0,0 +1,23 @@
+export type RetryCallbackProps = {
+  attempt: number;
+};
+
+export class Retry {
+  constructor(private maxRetries = 2, private retryDelay = 1000) {}
```

Comment:
You should extract `maxRetries` and `retryDelay` to CONSTANTS.

Reply:
`maxRetries` should be set from the number of `GITHUB_TOKEN`

Reply:
The helper does not care about the config, but who implements does.
importing constants will force mock others unused stuffs.

I will change the Retry call to pass the constants variables

Reply:
OK